packages/satellite/rubygem-foreman_theme_satellite/rubygem-foreman_theme_satellite.spec

```diff
@@ -0,0 +1,93 @@
+# template: foreman_plugin
+%global gem_name foreman_theme_satellite
+%global plugin_name theme_satellite
+%global foreman_min_version 3.7
+
+Name: rubygem-%{gem_name}
+Version: 13.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: This is a plugin that enables building a theme for Foreman
+License: GPLv3
+URL: https://github.com/RedHatSatellite/foreman_theme_satellite
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: ruby >= 2.7
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby >= 2.7
+BuildRequires: ruby >= 2.7
+BuildRequires: rubygems-devel
+BuildArch: noarch
+Provides: foreman-plugin-%{plugin_name} = %{version}
+BuildRequires: rubygem(activesupport)
+BuildRequires: rubygem(deface)
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
+# end package.json devDependencies BuildRequires
+
+# start package.json dependencies BuildRequires
+# end package.json dependencies BuildRequires
+
+%description
+Theme changes for Satellite 6.
+
+
+%package doc
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -q -n  %{gem_name}-%{version}
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build ../%{gem_name}-%{version}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%foreman_bundlerd_file
+%foreman_precompile_plugin -s
+
+%files
+%dir %{gem_instdir}
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/app
+%exclude %{gem_instdir}/config/metadata.yml
```

Comment:
Why is this excluded?

Reply:
it's only really needed during devel builds, for prod we read it from the satellite RPM that places it in a different path:
https://github.com/RedHatSatellite/foreman_theme_satellite/blob/develop/lib/foreman_theme_satellite/engine.rb#L132

Reply:
https://github.com/RedHatSatellite/foreman_theme_satellite/pull/9

Though when I completed it I realized it may have been sufficient to only exclude it in the gemspec. Still, I think this is cleaner.